packages/react-databrowser/src/components/databrowser/components/sidebar/db-size.tsx

```diff
@@ -0,0 +1,24 @@
+import { useDatabrowser } from "@/store"
+import { useQuery } from "@tanstack/react-query"
+
+import { formatNumber } from "@/lib/utils"
+import { Skeleton } from "@/components/ui/skeleton"
+
+export const DisplayDbSize = () => {
+  const { redis } = useDatabrowser()
+  const { data: keyCount } = useQuery({
+    queryKey: ["useFetchDbSize"],
```

Comment:
I guess we need to invalidate this query when we insert a key. Or maybe we can refetch this with an interval?